docs/modules/ROOT/pages/operations/browser-sso.adoc

```diff
@@ -27,102 +27,123 @@ It is strongly advised to use PKCE to ensure security.
-			"params": {  // can be used for both the auth and the token request
-				"client_id": "account",
-				"redirect_uri": "http://<browser-server-host>:<http-port>?idp_id=keycloak-oidc&auth_flow_step=redirect_uri",
-				"response_type": "code",  // depends on the auth_flow
-				"scope": "openid groups"
-			},
-			"auth_params": { // optional
-				"param_p": "<extra parameter used only for the auth request>"
-			},
-			"token_params": { // optional
-				"client_secret": "<secret-here>", // this may be required by some Idp's and depended on the auth flow.
-				"param_p": "<extra parameter used only for the token request>"
-			},
-			"config": { // optional settings, these allow you to overwrite the defaults
-				"implicit_flow_requires_nonce": false, // Default: false; Desc: Specify if the implicit auth flow requries a nonce in the request
-				"principal": "preferred_username",  // Default: email, otherwise sub; Desc: Optional, in which token claim the user's principal is specified
-				"token_type_principal": "access_token" // Default: access_token; Desc: Which token type is decoded to acquire the specified principal
-				"token_type_authentication": "access_token" // Default: access_token; Desc: Which token type is used as password
-				"code_challenge_method": "S256" // Default is "S256" and it's the only supported method at this moment.
-			}
-		}
-	]
-}
-----
-+
 [IMPORTANT]
 ====
-`redirect_uri` MUST match exactly the redirect_uri specified in the IdP.
+Security information should always be exchanged with encrypted transport, and therefore HTTPS should be used.
+Mixed HTTP/HTTPS flows for single sign-on are not supported.
 ====
 
+Once your SSO provider is configured, you need to configure Neo4j to use OpenID Connect.
+This is done by updating the link:https://neo4j.com/docs/operations-manual/current/configuration/neo4j-conf/#neo4j-conf[`neo4j.conf`^] file according to the instructions in link:https://neo4j.com/docs/operations-manual/current/authentication-authorization/sso-integration/#auth-sso-configure-sso[Operations Manual -> Configure Neo4j to use OpenId Connect^].
+
+Make sure to avoid duplicate entries in the `neo4j.conf` file.
+
+Browser needs to be aware of the identity providers available for use.
+When used with Neo4j v4.4+, this is specified the `neo4j.conf` file, as described above.
+
+Deployments that use an earlier version of Neo4j require a separate OAuth plugin and assistance from Professional Services, as mentioned previously.
+However, in such cases, the identity providers can be specified by a URL parameter `discoveryURL` that defines a URL to a json file containing the SSO providers.
+Example for Browser: `\https://<browser-server-host>:<http-port>?discoveryURL=https://webhost.com/public/discovery.json`
+
+[IMPORTANT]
+====
+Browser will prioritize in the following order, in case of conflicting data:
+. `neo4j.conf`
+. `discoveryURL`
```

Comment:
What data gets prioritized is a bit more complicated than this, we check a total of five sources and prioritize them in the following order:
- neo4j.conf data from the server entered in the :server connect form 
- neo4j.conf data from the server located at `connectURL` url param
- discoveryURL data
- neo4j.conf data from the server previously connected to (if any) 
- neo4j.conf data from the server that hosts browser

All data will be merged, given that the "host" field matches. Not sure what the best way to present this is, for the end user what it comes down to is -> point neo4j browser towards a 4.4 server and it'll use the config from that server. If you want to use some other config, use `discoveryURL`